src/renderer/components/+nodes/node-charts.tsx

```diff
@@ -17,26 +18,26 @@ export const NodeCharts = observer(() => {
   const id = object.getId();
   const { chartCapacityColor } = themeStore.activeTheme.colors;
 
-  if (!metrics) return null;
-  if (isMetricsEmpty(metrics)) return <NoMetrics/>;
+  if (!metrics) {
+    return null;
+  }
 
-  const values = Object.values(metrics).map(metric =>
```

Comment:
Why it's brittle? And what the reason to use `mapValues` from `lodash`? 
If iterating only values not enough I would use `Object.entries(metrics)` instead..

Reply:
It is brittle because it depends on the order that the fields were created. Which I don't believe that we guarantee anywhere.

As for using `mapValues`. It is much shorter, does what I want, and we are already using `lodash` so I don't think the argument about an extra helper holds water.